releasenotes/notes/1.2/linear-binary-matrix-utils-rust-c48b5577749c34ab.yaml

```diff
@@ -2,7 +2,7 @@
 features_synthesis:
   - |
     Port internal binary matrix utils from Python to Rust, including
-    binary matrix multiplication, gaussian elimination, rank calculation,
+    binary matrix multiplication, Gaussian elimination, rank calculation,
     binary matrix inversion, and random invertible binary matrix generation.
-    These functions are not part of the Qiskit API, and porting them to rust
+    These functions are not part of the Qiskit API, and porting them to Rust
     improves the performance of certain synthesis methods.
```

Comment:
I'm not sure we need a release note for this since it's not user facing. I'd expect us to have a release note for the user facing synthesis function relying on these internal utility functions.